packages/expo-image/src/Image.types.ts

```diff
@@ -31,6 +31,13 @@ interface AndroidGlideException extends AndroidThrowable {
   causes: AndroidThrowable[] | null;
 }
 
+export interface ImageResolvedAssetSource {
+  height: number;
+  width: number;
+  scale: number;
+  uri: string;
+}
+
```

Comment:
```suggestion
```

It is fine this way, however in situation when we are exporting RN method I would consider also exporting RN interface (as long as it is exact copy of it). Up to you :smile:

Reply:
I also considered it, but I can't decide which solution is better. 😅😅